types/decentlab/IAM/meta.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "IAM",
+  "name": "DL-IAM",
```

Comment:
please note that name changes are breaking - therefore let's revert this if it is not needed

Reply:
Smolo wanted it ^^

Reply:
ok call me before merging so that we can coordinate this change